project/dexter-vs/Dexter/PeerReview/Core/PeerReviewSnapshotComment.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Text;
+using Dexter.PeerReview.Utils;
+
+namespace Dexter.PeerReview
+{
+    /// <summary>
+    /// Provides review comment in SnapshotSapn
+    /// </summary>
+    public class PeerReviewSnapshotComment : PeerReviewComment
+    {
+        SnapshotSpan snapshotSpan;
+        
+        /// <summary>
+        /// SnapshotSpan contains review comment
+        /// </summary>
+        public SnapshotSpan SnapShotSpan
+        {
+            get { return snapshotSpan; }
```

Comment:
You can use auto-property here:

public SnapshotSpan SnapshotSpan
{
   get;
   private set;
}